pkg/ccl/backupccl/restore_data_processor.go

```diff
@@ -401,12 +394,11 @@ func (rd *restoreDataProcessor) processRestoreSpanEntry(
 	iter := sst.iter
 	defer sst.cleanup()
 
-	writeAtBatchTS := restoreAtNow.Get(&evalCtx.Settings.SV)
-
 	// If the system tenant is restoring a guest tenant span, we don't want to
 	// forward all the restored data to now, as there may be importing tables in
 	// that span, that depend on the difference in timestamps on restored existing
 	// vs importing keys to rollback.
+	writeAtBatchTS := true
```

Comment:
nit: do we need this var now?

Reply:
Yes, see branch below -- it's disabled for tenant restores.

Reply:
doh yes, thanks!